src/Tools/BuildValidator/BuildValidator.csproj

```diff
@@ -4,7 +4,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net472;net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net472</TargetFrameworks>
```

Comment:
This change looks intentional. What's the purpose?

Reply:
Visual Studio debugging of .NET Framework projects is broken on my machine hence I moved `net6.0` first because .NET Core debugging is not.